src/client/pages/Payment/Payment.tsx

```diff
@@ -22,106 +25,162 @@ export const PaymentPage = () => {
   }
 
+  if (showSuccess) {
+    return (
+      <Centered>
+        <TickIcon size="4rem" />
+        <Text>{textKeys.AFTER_SIGN_PAYMENT_SUCCESS_MESSAGE()}</Text>
+      </Centered>
+    )
+  }
+
   return (
     <SessionTokenGuard>
-      <Wrapper>
+      <Wrapper y={{ base: 1.5, largeScreen: 2.5 }}>
         <PageHeader>
-          <HedvigLogo width={82} />
-
-          <HeaderLink
-            href={CUSTOMER_SERVIVE_URL[pathLocale]}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            {textKeys.AFTER_SIGN_PAYMENT_HELP_LINK()}
-          </HeaderLink>
+          <HeaderLogo>
+            <HedvigLogo width={78} />
+          </HeaderLogo>
+          <HeaderBreadcrumbs>
+            <Breadcrumbs />
+          </HeaderBreadcrumbs>
+          <HeaderBack>
+            <HeaderLink
+              href={CUSTOMER_SERVIVE_URL[pathLocale]}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {textKeys.AFTER_SIGN_PAYMENT_HELP_LINK()}
+            </HeaderLink>
+          </HeaderBack>
         </PageHeader>
-        <Header>
-          <Heading>{textKeys.ONBOARDING_CONNECT_DD_CTA()}</Heading>
-          <Paragraph>{textKeys.AFTER_SIGN_PAYMENT_MESSAGE()}</Paragraph>
-        </Header>
-        {!redirectURL ? (
-          <Paragraph>{textKeys.GENERIC_ERROR_HEADING()}</Paragraph>
-        ) : (
-          trustlyURL && (
-            <TrustlyIframe
-              url={trustlyURL}
-              onSuccess={() => handleSuccess(redirectURL)}
-              onFail={() => handleFail(redirectURL)}
-            />
-          )
-        )}
+
+        <Content y={{ base: 2, largeScreen: 3 }}>
+          <LargeParagraph>
+            {textKeys.AFTER_SIGN_PAYMENT_MESSAGE()}
+          </LargeParagraph>
+          {!redirectURL ? (
+            <SmallParagraph>{textKeys.GENERIC_ERROR_HEADING()}</SmallParagraph>
+          ) : showSuccess ? (
+            <SmallParagraph>Hello!</SmallParagraph>
+          ) : (
+            trustlyURL && (
+              <Space y={0.75}>
+                <TrustlyIframe
+                  url={trustlyURL}
+                  onSuccess={() => handleSuccess(redirectURL)}
+                  onFail={() => handleFail(redirectURL)}
+                />
+                <SmallParagraph>
+                  {textKeys.AFTER_SIGN_PAYMENT_FOOTNOTE()}
+                </SmallParagraph>
+              </Space>
+            )
+          )}
+        </Content>
       </Wrapper>
     </SessionTokenGuard>
   )
 }
 
-const Wrapper = styled.div({
-  display: 'flex',
-  flexDirection: 'column',
-  gap: '2rem',
+const Wrapper = styled(Space)({
   paddingBottom: '2rem',
+  paddingInline: '1rem',
 })
 
-const PageHeader = styled.div({
-  height: '3.5rem',
-  paddingInline: '1rem',
-  display: 'flex',
-  justifyContent: 'space-between',
+const LargeParagraph = styled.p({
+  margin: 0,
+  fontSize: '1.5rem',
+  lineHeight: 1.26,
```

Comment:
nit: quite specific number :D did we mean 1.25?

Reply:
I picked it from Figma 😅